### ISSUE TITLE
Mouse clicks unrealiable on Raspberry Pi.

### ISSUE THREAD

**Issue by JORGETECH** [ioquake/ioq3#389]
I wanted to try out Quake III Arena on my Raspberry Pi so I built ioquake3 from git and after setting the OpenGL version to 1 it works great!

However there seems to be a weird issue with how the mouse works, sometimes I click the mouse button but it doesn't fire and other times it starts firing automatically. I tried changing the "in_mouse" value with no luck.

**Comment by ensiform** [comment]
in_mouse values have no effect with SDL2 iirc.

What version of SDL is being used?  Likely a good possibility of a SDL<>RPI bug.

**Comment by JORGETECH** [comment]
SDL version 2.0.5+dfsg1-2 is being used.

**Comment by ensiform** [comment]
Can you try upgrading to newest build? 2.0.8

**Comment by cdev-tux** [comment]
SDL 2.0.9 is out now. I know they fixed a different issue with the mouse on the Pi, so it might help.

**Comment by JORGETECH** [comment]
I could upgrade indeed but Raspbian is using Debian Stretch packages that are not likely to be updated. Is it recommended to build Debian packages from the Sid sources?

**Comment by cdev-tux** [comment]
Yes, compile SDL 2.0.9 from source, and then build ioquake3 with the new SDL libraries to see if that solves the issue.

**Comment by JORGETECH** [comment]
Is there any environment flag for the SDL source directory?

**Comment by NuclearMonster** [comment]
Sorry this issue is so old, could anyone verify it's still a problem with the raspberry pi with whatever the latest OS is?
